src/core/pure-js/deck-js.js

```diff
@@ -163,6 +168,17 @@ export default class DeckGLJS {
       this.layerManager.finalize();
       this.layerManager = null;
     }
+
+    if (this._conroller) {
+      this._controller.finalize();
+      this._controller = null;
+    }
+  }
+
+  // Trigger transition, retur true if a new transition is triggered, false otherwise.
+  triggerViewportTransition(nextProps) {
```

Comment:
Naming? This sound like an action, not a predicate: how about `isViewportTransitionTriggered`?